package/MDAnalysis/core/groups.py

```diff
@@ -317,9 +317,94 @@ class _ImmutableBase(object):
+    |                               |            | from ``s`` and from ``t``  |
+    +-------------------------------+------------+----------------------------+
+    | ``s.subtract(t)``             | ``s - t``  | new Group with elements    |
+    |                               |            | from ``s`` that are not    |
+    |                               |            | in ``t``                   |
+    +-------------------------------+------------+----------------------------+
+
+    The following operations treat the Group as set. Any result will have any
+    duplicate entries removed and the Group will be sorted.
+
+    +-------------------------------+------------+----------------------------+
+    | Operation                     | Equivalent | Result                     |
+    +===============================+============+============================+
+    | ``s.isdisjoint(t)``           |            | ``True`` if ``s`` and      |
+    |                               |            | ``t`` do not share         |
+    |                               |            | elements                   |
+    +-------------------------------+------------+----------------------------+
+    | ``s.issubset(t)``             |            | test if all elements of    |
+    |                               |            | ``s`` are part of ``t``    |
+    +-------------------------------+------------+----------------------------+
+    | ``s.is_strict_subset(t)``     |            | test if all elements of    |
+    |                               |            | ``s`` are part of ``t``,   |
+    |                               |            | and ``s != t``             |
+    +-------------------------------+------------+----------------------------+
+    | ``s.issuperset(t)``           |            | test if all elements of    |
+    |                               |            | ``t`` are part of ``s``    |
+    +-------------------------------+------------+----------------------------+
+    | ``s.is_strict_superset(t)``   |            | test if all elements of    |
+    |                               |            | ``t`` are part of ``s``,   |
+    |                               |            | and ``s != t``             |
+    +-------------------------------+------------+----------------------------+
+    | ``s.union(t)``                | ``s | t``  | new Group with elements    |
+    |                               |            | from both ``s`` and ``t``  |
+    +-------------------------------+------------+----------------------------+
+    | ``s.intersection(t)``         | ``s & t``  | new Group with elements    |
+    |                               |            | common to ``s`` and ``t``  |
+    +-------------------------------+------------+----------------------------+
+    | ``s.difference(t)``           |            | new Group with elements of |
+    |                               |            | ``s`` that are not in ``t``|
```

Comment:
Nope, currently its:
```python
ag1 = u.atoms[[1, 2, 1, 2, 3, 4]]
ag2 = u.atoms[[3, 4, 5]]

ag1 + ag2 == [1, 2, 1, 2, 3, 4, 3, 4, 5]
ag1 & ag2 == [3, 4]
ag1 | ag2 == [1, 2, 3, 4, 5]
ag1 - ag2 == [1, 2, 1, 2]  # ie take from ag1 unless in ag2

```

@jbarnoud Maybe we should make `__sub__ == difference` and leave subtract as method only access?